formula/src/main/java/com/instacart/formula/internal/ListInspector.kt

```diff
@@ -0,0 +1,69 @@
+package com.instacart.formula.internal
+
+import com.instacart.formula.DeferredAction
+import com.instacart.formula.Inspector
+import kotlin.reflect.KClass
+
+internal class ListInspector(
+    private val inspectors: List<Inspector>,
+): Inspector {
+    override fun onFormulaStarted(formulaType: KClass<*>) {
+        for (inspector in inspectors) {
+            inspector.onFormulaStarted(formulaType)
+        }
```

Comment:
nit: maybe introduce `private fun forEveryInspector(block: Inspector.() -> Unit)` to make this class less verbose and a little more kt idiomatic?

```
forEveryInspector { onFormulaStarted(formulaType) }
```